Further changes to AAP on GCP doc (#28)
ansible/aap-docs: stories/topics/ref-aap-gcp-automation-performance.adoc

[cols="30%,30%",options="header"]
|====
| Metric | Threshold
| Concurrent Jobs | 10
| Forks per job | 10
|====

[NOTE]
====

> {AAPonGCP} is driven using three m5.large instances, two of which run {PlatformName} {ControllerName} and one which runs {HubName}. The {ControllerName} instances collectively support a standard workload for 100 managed active nodes. 

Reviewer: Do we need to format `m5.large` ?

Author: I've not seen it formatted before.

> Testing this operating criteria by Red Hat found support for up-to 10 forks each.  

Reviewer: Is the dash in "up-to" necessary?

> Testing this operating criteria by Red Hat found support for up-to 10 forks each.  

Reviewer: ```suggestion
Red Hat has tested the operating criteria for up-to 10 forks each.  
```
Suggesting that this be re-phrased.  My attempt could be misinterpreting the sentence, though.

> This operating criteria above has been set and tested using output-intensive, “chatty” workloads that produce 2 messages 7 seconds apart on both {ControllerName} nodes. 

Reviewer: ```suggestion
This operating criteria has been set and tested using output-intensive, “chatty” workloads that produce 2 messages 7 seconds apart on both {ControllerName} nodes. 
```
